README.md

```diff
@@ -15,8 +15,9 @@ This chart will deploy the following on the target OpenShift cluster:
 - [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/docs/getting-started/) has been installed in your cluster.
 - Your cluster has a [default storage class](https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/persistent-storage-csi-sc-manage.html) provisioned.
 - [Helm](https://helm.sh/docs/intro/install/) v3.9+ is installed.
-- [PostgreSQL](https://www.postgresql.org/) database is avalable with credentials to manage the tablespace.
+- [PostgreSQL](https://www.postgresql.org/) database is avalable with credentials to manage the tablespace (optional).
   - A [reference implementation](#postgresql-deployment-reference-implementation) is provided for your convenience.
+- A Github API Token - in order to import items into the catalog, there is a need for GITHUB_TOKEN with the permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/).
```

Comment:
When I used this page to generate the token, it did not work, may be it's worth mentioning that we shall use the fine grain one

Reply:
I updated the section for classic token. Tested it, it worked.